### PR TITLE
[chore] 修正 generate-icon-component script

### PR DESCRIPTION
**Pull request by benny0642**
# Purpose

- Asana link: https://app.asana.com/0/347798529122928/1202003234808015/f
- 由於 svgr/cli 有跳大版號（v5 -> v6），componentTemplate 吃的參數有變，所以才會壞掉，解法是限制版號在 5.5.0 即可。
- 另外由於 copy-link, add-location, location, radio-empty, radio-selected 這幾個圖檔有被改到，所以需要測試

截圖 2022-05-06 下午4 24 35

# Changes

- a list of what have been done
- maybe some code change

# Risk

Usually none, if you have any please write it here.

# TODOs

- [ ] Describe what should be done outside of this PR
- [ ] Maybe in other PRs or some manual actions.

### CODE REVIEW

**Review by chenesan — APPROVED** [review]

**Review by tz5514 — APPROVED** [review]

File: packages/core/src/icons/components/CopyLink.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default function SvgCopyLink(props) {
       </g>
       <defs>
         <clipPath id="copy-link_svg__clip0">
-          <path fill="transparent" transform="translate(8 8)" d="M0 0h16v16H0z" />
+          <path fill="#fff" transform="translate(8 8)" d="M0 0h16v16H0z" />
```
Comment (leannechen) [inline packages/core/src/icons/components/CopyLink.js:19]: 這是改成白色以後，下 color 背景色還正常嗎？

**Review by leannechen — APPROVED** [review]
👍🏼

File: packages/core/src/icons/components/CopyLink.js (hunk 1 above)
Comment (benny0642) [reply packages/core/src/icons/components/CopyLink.js:19]: 正常，我測試過了

Comment (leannechen) [reply packages/core/src/icons/components/CopyLink.js:19]: 好～感謝

**Pull request merged**